dnd5esheets/front/src/store/index.ts

```diff
@@ -1,491 +1,358 @@
-import { createComputed } from 'solid-js'
-import { createStore, reconcile } from 'solid-js/store'
-import { CharacterSchema } from '~/5esheets-client'
-import { ActionType } from '~/5esheets-client'
+import {
+  Resource,
+  createComputed,
+  createResource,
+  createReaction,
+  createMemo,
+  createEffect,
+  Accessor,
+} from 'solid-js'
+import {
+  SetStoreFunction,
+  Store,
+  createStore,
+  produce,
+  reconcile,
+  unwrap,
+} from 'solid-js/store'
+import { createRouteData } from 'solid-start'
+import { CharacterSchema, ListCharacterSchema } from '~/5esheets-client'
 import { Proficiency } from '~/5esheets-client'
 import { Scores } from '~/5esheets-client'
 import { SaveProficiencies } from '~/5esheets-client'
 import { SkillProficiencies } from '~/5esheets-client'
+import { CharacterService } from '~/5esheets-client'
+import { applyEffects, computeEffect, Effect } from '~/effects'
+import { AssignmentOperator } from '~/effects/parser'
```

Comment:
I removed un-used imports

```suggestion
import {
  Resource,
  createResource,
  createMemo,
  createEffect,
} from 'solid-js'
import {
  createStore,
  produce,
} from 'solid-js/store'
import { CharacterSchema, ListCharacterSchema } from '~/5esheets-client'
import { applyEffects, computeEffect } from '~/effects'
```